CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [v0.7.1] - 2021-01-25
```

Comment:
I don't think we should release a 0.7.1 which doesn't list any signs of motivitation or changes. Are we going to yank 0.7.0? If so that is even more important...

Reply:
I don't understand what you mean exactly, the changelog contains everything that was previously "unreleased", i.e. has a bunch of added, deprecated, and fixed entries. What else should we include?

Reply:
Hm, weird; seems I was holding the PR wrong. Now I see it. 😅

Reply:
Okay, I was totally missing that #319 exists and was merged and the way the collapsed diff rendered led me to the conclusion that the changes were not there. Sorry about that.